src/components/icon/demoBasicUsage/index.html

```diff
@@ -0,0 +1,57 @@
+
+<div class="container" ng-controller="IconCtrl" ng-cloak>
+  <div class="loading" ng-show="loading">
+    <md-progress-circular class="md-accent" md-mode="indeterminate"></md-progress-circular>
+  </div>
+  <header>
+    <h1>Material: md-icon</h1>
+  </header>
+    <md-radio-group ng-model="iconType">
+      <label>Icon Type: </label>
+      <md-radio-button value="SVG" aria-label="SVG">SVG</md-radio-button>
+      <md-radio-button value="Font" aria-label="Font">Font</md-radio-button>
+    </md-radio-group>
+    <md-input-container flex>
+      <label>Color</label>
+      <input ng-model="color">
+    </md-input-container>
+
+    <div class="glyph" ng-repeat="icon in icons | limitTo:limit" ng-if="iconType == 'SVG'">
+      <div class="preview-glyphs">
+        <md-icon ng-repeat="size in sizes" svg-icon="{{ icon.name }}"
+                 style="fill:{{ color }};width:{{ size }}px;height:{{ size }}px;" alt="{{ icon.name + size }}">
```

Comment:
@ThomasBurleson OSS is all about everyone contributing at every point. I don't know why you wouldn't want suggestions and feedback from everyone before code has released...that prevents bugs like what would have happened if you released this code with `const`. I'm just saying that you guys really need to let down on the restrictions and stuff and start just letting everyone contribute at any point and encouraging that.